pkg/controller/stack/stack_controller.go

```diff
@@ -19,7 +19,6 @@ import (
 	"time"
```

Comment:
NB: most logic added in this file determines how/when a Stack CR should be reconciled.